project-base/scripts/configure.sh

```diff
@@ -37,9 +37,7 @@ if [[ -d "project-base" ]]; then
     echo "You are in monorepo, prefixing paths app paths with ${projectPathPrefix}"
 fi
 
-echo "Creating config files.."
-cp -f "${projectPathPrefix}config/parameters.yaml.dist" "${projectPathPrefix}config/parameters.yaml"
-cp -f "${projectPathPrefix}config/parameters_test.yaml.dist" "${projectPathPrefix}config/parameters_test.yaml"
+echo "Creating domain file.."
```

Comment:
nitpick, but I would be rather more specific...
```diff
- echo "Creating domain file.."
+ echo "Creating domains_urls.yaml file.."
```

Reply:
823ee2eb3 it is